app/Livewire/Project/Database/BackupEdit.php

```
@@ -82,6 +84,58 @@ class BackupEdit extends Component
     #[Validate(['required', 'int', 'min:60', 'max:36000'])]
     public int $timeout = 3600;
 
+    #[Validate(['required', 'string'])]
+    public string $engine = 'native';
+
+    #[Validate(['nullable', 'string', 'in:full,diff,incr'])]
+    public ?string $pgbackrestBackupType = 'full';
+
+    #[Validate(['nullable', 'string', 'in:none,bz2,gz,lz4,zst'])]
+    public ?string $pgbackrestCompressType = 'lz4';
```

PHPMD (2.15.0)

Avoid excessively long variable names like $pgbackrestCompressType. Keep variable name length under 20. (undefined)

(LongVariable)